taxi/views.py

```diff
@@ -87,3 +90,31 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreationForm
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    form_class = DriverLicenseUpdateForm
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+@login_required()
+def assign_to_car(request, pk):
+    driver = Driver.objects.get(id=request.user.id)
+    car = Car.objects.get(id=pk)
+    if car in driver.cars.all():
```

Comment:
use .exists() so that db will check it using EXISTS and WHERE, because in this way you make query to db, it fetches all records and the python loops through all objects to check if car is in this list